### DISCUSSION TITLE
QR Code

### DISCUSSION THREAD

**Discussion by MrPatrickHenry** [Attendize/Attendize#1100] [General]
Hello I have been using Attendize for years, now club I use it for want to use the QR feature never needed to test it before but the code seems to generate just a string is there something I am missing, a URL to use or to post it to or to amend the qr generation to a link? 
any pointers be great thank you

**Comment by MrPatrickHenry** [comment]
i figured it out in the routes file, btw I love that feature worked amazingly well.

**Reply by AlgorithmGG** [reply]
Are you using a saas version?

**Reply by MrPatrickHenry** [reply]
no self hosted. i am good now. i would like the to be a diffetn user view, I don't need the guys at the front seeing all the numbers.
